aws-sns-lambda/variables.tf

```diff
@@ -0,0 +1,92 @@
+variable project {
+  type        = string
+  description = "Project for tagging and naming."
+}
+
+variable env {
+  type        = string
+  description = "Env for tagging and naming."
+}
+
+variable service {
+  type        = string
+  description = "Service for tagging and naming"
+}
+
+variable owner {
+  type        = string
+  description = "Owner for tagging and naming."
+}
+
+variable lambda_environment {
```

Comment:
I think this might be easier if instead of taking all the lambda parameters and passing them through, it takes a `lambda_arn`.

Reply:
Have been thinking about this, that will be simple, but the downside is that would assume SNS to subscribe to whatever, which made it less integrative.